storagemodule.py

```diff
@@ -0,0 +1,57 @@
+#Question: Do you want me to make this whole thing a module or lots of submodules ? 
```

Comment:
I think let's go with one module for now. The idea of a module is that it contains one idea. If this module is meant to handle the storage folder lifecycle then creation deletion editing etc... should be contained in it. Also you can delete this comment now :)